scripts/pre_push_hook.py

```diff
@@ -60,7 +60,9 @@
 FILE_DIR: Final = os.path.abspath(os.path.dirname(__file__))
 OPPIA_DIR: Final = os.path.join(FILE_DIR, os.pardir, os.pardir)
 LINTER_FILE_FLAG: Final = '--files'
-PYTHON_CMD: Final = 'python'
+# Path to currently running python interpreter,
+# it is required to resolve python version conflict in docker.
+PYTHON_CMD: Final = sys.executable
```

Comment:
This can break the python setup. Using the `OPPIA_IS_DOCKERIZED` variable change the value

Reply:
done!